### PR TITLE
Create CR2 publishing package (#1220).

### LINKED ISSUE #1220: Create CR2 package for publishing.
Per [1], prepare CR2 publishing package, tentatively using March 2, 2021 as publishing date.

[1] https://www.w3.org/2021/02/18-tt-minutes.html

### PR DESCRIPTION
**Pull request by skynavga**
Closes #1220.

### CODE REVIEW

File: spec/status.xml [hunk 1]
```diff
@@ -6,9 +6,12 @@
...
 the <loc href="https://www.w3.org/TR/">W3C technical reports index</loc> at https://www.w3.org/TR/.</emph></p>
 
-<p>This is the First <loc href="https://www.w3.org/2019/Process-20190301/#candidate-rec">Candidate Recommendation</loc> (CR1) of
+<p>This is the Second <loc href="https://www.w3.org/2019/Process-20190301/#candidate-rec">Candidate Recommendation</loc> (CR2) of
 the Second Edition of the Timed Text Markup Language &versionOfTT; (TTML&versionOfTT;), and is expected to advance to Proposed
-Recommendation no earlier than 17 March 2020.</p>
+Recommendation no earlier than 30 March 2021.
+This Second <loc href="https://www.w3.org/2019/Process-20190301/#candidate-rec">Candidate Recommendation</loc> (CR2) makes no
+technical changes to the previous, First <loc href="https://www.w3.org/2019/Process-20190301/#candidate-rec">Candidate
+Recommendation</loc> (CR1); that is, only editorial (non-substantive) changes were applied.</p>
```
Comment (nigelmegitt) [inline spec/status.xml:14]: Do we need to state that the applicable Process version has changed?

Comment (himorin) [reply spec/status.xml:14]: Need to change link to the Process 2020 (as in our charter), but I don't think we need to state something on this change.

**Comment by himorin** [comment]
We need to have a section of change from CR1 to CR2 at somewhere. Could you add to ttml2-changes, like [one in css-background-3](https://drafts.csswg.org/css-backgrounds-3/#changes) - even if all editorial.

File: spec/status.xml (hunk 1 above)
Comment (skynavga) [reply spec/status.xml:14]: Yes, we will need to change in SOTD. I determined this after running pubrules, and noted it is applying the new process.

**Comment by skynavga** [comment]
@himorin I will add.

File: spec/status.xml (hunk 1 above)
Comment (skynavga) [reply spec/status.xml:14]: Fixed

**Comment by skynavga** [comment]
Updated publishing package to fix pubrule issues; updated ttml2-changes.html to include CR1 to CR2 changes.

**Comment by himorin** [comment]
Thank you for your heavy work on updating the package!
I'll try to run pubrules etc. asap and will back with results and update transition request, with target 5th Mar - next slot as this Fri.

**Comment by himorin** [comment]
@skynavga Updated [transition request](https://github.com/w3c/transitions/issues/316). I hope to be approved in a slot this week, and actual publication @ Mar 9th. (due date for comments to be Apr 6)

**Comment by himorin** [comment]
I've pushed to /TR/ with following changes

```
diff --git a/spec/entitiescr.dtd b/spec/entitiescr.dtd
index 586a7162..18de9fd1 100644
--- a/spec/entitiescr.dtd
+++ b/spec/entitiescr.dtd
@@ -4,8 +4,8 @@
 <!ENTITY draft.year "2021">
 <!ENTITY draft.month "March">
 <!ENTITY draft.mm "03">
-<!ENTITY draft.day "02">
-<!ENTITY draft.dd "02">
+<!ENTITY draft.day "09">
+<!ENTITY draft.dd "09">
 
 <!ENTITY draft.date "&draft.year;&draft.mm;&draft.dd;">
 
diff --git a/spec/status.xml b/spec/status.xml
index f1e2ec30..2de09214 100644
--- a/spec/status.xml
+++ b/spec/status.xml
@@ -8,7 +8,7 @@ the <loc href="https://www.w3.org/TR/">W3C technical reports index</loc> at http
 
 <p>This is the Second <loc href="https://www.w3.org/2020/Process-20200915/#candidate-rec">Candidate Recommendation</loc> (CR2) of
 the Second Edition of the Timed Text Markup Language &versionOfTT; (TTML&versionOfTT;), and is expected to advance to Proposed
-Recommendation no earlier than 30 March 2021.
+Recommendation no earlier than 6 April 2021.
 This Second <loc href="https://www.w3.org/2020/Process-20200915/#candidate-rec">Candidate Recommendation</loc> (CR2) makes no
 technical changes to the previous, First <loc href="https://www.w3.org/2020/Process-20200915/#candidate-rec">Candidate
 Recommendation</loc> (CR1); that is, only editorial (non-substantive) changes were applied.</p>
```

**Review by himorin — APPROVED** [review]

**Pull request merged**